index.bs

```diff
@@ -3402,6 +3392,19 @@ Maturity Stages on the Recommendation Track</h4>
 			The W3C Royalty-Free IPR licenses
 			granted under the W3C Patent Policy [[PATENT-POLICY]]
 			apply to [=W3C Recommendations=].
+
+			<p id="expandable-rec">
+			After its initial publication,
+			a [=W3C Recommendation=] <em class=rfc2119>may</em> be [[#revising-rec|revised]]
+			to address [=editorial change|editorial=] or [=substantive change|substantive=] issues
+			that are discovered later.
+			However, <a href=#class-4>new features</a> can only be added
+			if the document already identifies itself
+			as intending to <dfn>allow new features</dfn>.
+			Such an allowance cannot be added
+			to a [=technical report=] previously published as a [=Recommendation=]
+			that did not allow such changes.
```

Comment:
```suggestion
			that did not allow such changes; this requires a new FPWD.
```

Reply:
```suggestion
			that did not allow such changes;
			this requires a new [=technical report=], that could for example be similarly named but with an incremented version number.
```

Reply:
I might end at the comma, but otherwise lgtm. :)